server/services/src/main/java/org/zanata/adapter/TSAdapter.java

```diff
@@ -252,6 +256,19 @@ private boolean usable(ContentState state, boolean approvedOnly) {
                 (!approvedOnly && state.isTranslated());
     }
 
+    /**
+     * Replace the language="en_US" with a target localeId
+     * @param event the DocumentPart event of a TS file header
+     * @param localeId the desired target locale
+     * @return a GenericSkeleton with the language replaced
+     */
+    private ISkeleton replaceLocaleInDocPart(Event event, String localeId) {
+        DocumentPart part = event.getDocumentPart();
+        return new GenericSkeleton(part.getSkeleton().clone().toString()
+                .replaceAll("language\\s*=\\s*\".+\"",
```

Comment:
This regex could mess with the sourcelanguage attribute. Also, it would be a bit safer to use `[^"]` instead of `.` between the quotes.

http://doc.qt.io/archives/qt-4.8/linguist-ts-file-format.html